pygsti/modelmembers/states/tpstate.py

```diff
@@ -158,6 +160,17 @@ def from_vector(self, v, close=False, dirty_value=True):
         self._ptr_has_changed()
         self.dirty = dirty_value
 
+    def stateless_data(self) -> Tuple[int]:
+        return (self.dim,)
+
+    @staticmethod
+    def torch_base(sd: Tuple[int], t_param: _Torchable.Tensor) -> _Torchable.Tensor:
+        torch = _Torchable.torch_handle
+        dim = sd[0]
+        t_const = (dim ** -0.25) * torch.ones(1, dtype=torch.double) 
```

Comment:
@coreyostrove, @sserita: same question as above. Is the scale factor of ``dim ** -0.25`` appropriate for general spaces?

Reply:
From dev meeting:

Docs for the ``__init__`` function mention an assumption on the nature of the basis associated to this TPState. That function should be changed to explicitly assert that the identity matrix is an element of the basis and is in fact the first element of the basis.

https://github.com/sandialabs/pyGSTi/blob/1ec6909a7f9149c4f6758f9cd46256dbbff984c1/pygsti/modelmembers/states/tpstate.py#L60-L68

Make the same change for TPPOVM and FullTPOp.